pytorch_lightning/trainer/connectors/logger_connector/result.py

```diff
@@ -260,6 +260,7 @@ def __getstate__(self, drop_value: bool = False) -> dict:
             distributed_available=distributed_available
         ):
             d = {k: v for k, v in self.__dict__.items() if k not in skip}
+        d = {k: v for k, v in self.__dict__.items() if k not in skip}
```

Comment:
why add this?

Reply:
I was checking something, removed !

Reply:
We will have to remove the context manager. It shouldn't be there.